src/core/agent/services/keriaNotificationService.test.ts

```diff
@@ -1931,55 +1931,6 @@ describe("Long running operation tracker", () => {
     expect(operationPendingStorage.deleteById).toBeCalledTimes(1);
   });
 
-  test("Cannot create connection if the connection is already created", async () => {
```

Comment:
Can we keep the tests but just skip them? So we don't have to go looking for them or re-write?

Reply:
like test.skip